core/src/main/scala/org/apache/spark/deploy/Client.scala

```diff
@@ -61,6 +61,10 @@ private class ClientEndpoint(
 
    private val lostMasters = new HashSet[RpcAddress]
    private var activeMasterEndpoint: RpcEndpointRef = null
+   private val waitAppCompletion = conf.getBoolean("spark.standalone.submit.waitAppCompletion",
```

Comment:
Hm, do we use the standard parameters classes for these in this part of the code?

Reply:
There doesn't seem to be any config class specific to standalone deploy related code.

Reply:
We usually use `ConfigEntry` for a new conf. Could you add it too?

Reply:
@Ngone51 There doesn't seem to be any config class specific to standalone deploy related code. Could you kindly suggest where I should be adding this config?